js/hole-ng.js

```diff
@@ -176,25 +175,30 @@ const runCode = document.querySelector('#run a').onclick = async () => {
     status.style.background = data.Pass ? 'var(--green)' : 'var(--red)';
     statusH2.innerText      = data.Pass ? 'Pass 😀'      : 'Fail ☹️';
 
-    result = {
-        arguments: data.Argv.join('\n'),
-        diff:      data.Diff,
-        errors:    data.Err,
-        expected:  data.Exp,
-        output:    data.Out,
-    };
-
-    for (const div of statusDivs)
-        div.replaceChildren(new EditorView({
-            state: EditorState.create({
-                doc: result[div.id],
-                extensions: [
-                    ...baseExtensions,
-                    EditorView.editable.of(false),
-                    EditorView.lineWrapping,
-                    div.id == 'diff' ? extensions.diff : [],
-                ] }),
-        }).dom);
+    const argDiv = document.querySelector('#status > #arguments');
+    // Remove all arg spans
+    while (argDiv.firstChild)
+        argDiv.removeChild(argDiv.firstChild);
+
+    // Add a span for each arg
+    for(const arg of data.Argv) {
+        argDiv.appendChild(<span>{arg}</span>);
+        argDiv.appendChild(document.createTextNode(' '));
```

Comment:
At the very least we can do `argDiv.appendChild(<span>{arg}</span>, ' ')` to append both children at once, but that text node spacing it a little crummy, can we not just use `flexbox` and `gap`? If we no longer need the spacers at the markup level (which seems correct), we can produce a single list of all the nodes and then we can use `.replaceChildren(...data.Argv.map etc.)` to replace the above clearing too.